lib/optimizer_largetest.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# Copyright 2014 Google.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Integration tests that are targeted at the optimizer.
+# These are too large to be executed every iteration, but should be
+# executed before every checkin.
+#
+import random
+import re
+import unittest
+
+import encoder
+import optimizer
+import test_tools
+import vp8
+
+class DummyCodec(encoder.Codec):
```

Comment:
But the name "DummyCodec" is used throughout the testsuite, variation would be confusing. Sticking with consistency.